drivers/gpio/gpio_sifive.c

```diff
@@ -61,34 +61,80 @@ struct gpio_sifive_data {
 #define DEV_GPIO_DATA(dev)				\
 	((struct gpio_sifive_data *)(dev)->driver_data)
 
+/* _irq_level and _level2_irq are copied from
```

Comment:
Let's prefix this paragraph with `TODO:`.

Reply:
Added